Sources/MobileCoinClient.swift

```diff
@@ -204,6 +204,7 @@ public final class MobileCoinClient {
             metaFetcher: metaFetcher,
             txOutSelectionStrategy: txOutSelectionStrategy,
             mixinSelectionStrategy: mixinSelectionStrategy,
+            rng: MobileCoinDefaultRng(),
```

Comment:
thinking maybe its better to have one instance of the default RNG that we keep attached to the client.

Reply:
Ah, yeah, I think you're right.  When I switched to passing param to the `prepare` calls I pulled that out (I originally had it set up to be attached to the client when just getting things working).

Will update to default to using client's RNG (which will be MobileCoinDefaultRng()) when no rng is passed as a param

Reply:
updated.